pages/logout.js

```diff
@@ -0,0 +1,45 @@
+export default function Logout() {
```

Comment:
same, let's just create auth routes. No need  to add them as separate UI elements

Reply:
done